pkg/cmd/extension/command.go

```diff
@@ -441,16 +441,16 @@ func NewCmdExtension(f *cmdutil.Factory) *cobra.Command {
 
 					Press q to quit.
 
-					Running this command with --single-column should make this command
+					Running this command with %[1]s--single-column%[1]s should make this command
```

Comment:
I can't leave a comment due to change boundaries, but we should probably wrap `?` and `q`